tools/ResourceMetaSchema.json

```diff
@@ -204,6 +204,32 @@
         {
           "type": "object",
           "properties": {
+            "anyOf": {
+              "type": "array",
+              "items": {
+                "oneOf": [
```

Comment:
I actually copied this entry from the one above. I can update it if you don't like them. :)